productivity-suite/piercing-library/src/piercing-gateway.ts

```diff
@@ -368,6 +368,7 @@ export class PiercingGateway<Env> {
       <piercing-fragment-host fragment-id=${fragmentConfig.fragmentId}>
         ${prePiercingStyles}
         --FRAGMENT_CONTENT--
+        <div class="fragment-end"></div>
```

Comment:
I can use `body.show-seams piercing-fragment-host:has(*)` instead. Dario mentioned a corner case where the fragment host contains non-visible elements (which would still cause the seams to display). I can't repro that with the productivity suite, but it can definitely happen.

I can't really think of an alternative to the above selector that wouldn't involve embedding a sentinel value element that we then query for with a more specific selector. Open to ideas.

Personally, I vote to go with `body.show-seams piercing-fragment-host:has(*)` for now since it makes the demo code simpler unless we can find a use case within the productivity suite which breaks things.